README.md

```diff
@@ -70,3 +71,77 @@ Give a ⭐️ if you like this project!
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
```

Comment:
- [ ] @Abel-desu as the previous reviewer suggested you seem to have duplicated your readme file, from line one to 76 and again from 76 to 147, below is a small snippet of how it looks
![image](https://user-images.githubusercontent.com/61408860/194595522-66cfd2bf-3ea8-4d47-aade-17c5bb39f3df.png)

- [ ] [optional] The git clone link doesn't belong to this project it would be nice if you changed it.